broker-cli/commands/orderbook.js

```diff
@@ -58,10 +150,73 @@ function createUI (market, asks, bids) {
       'right': '',
       'right-mid': '',
       'middle': ''
+    },
+    colWidths: [COLUMN_WIDTHS, COLUMN_WIDTHS],
+    style: {
+      'padding-left': 0,
+      'padding-right': 0
+    }
+  })
+
+  gapTable.push([' ', ' '])
+
+  const displayAsks = (asks) => {
+    // Asks are added to the table with highest (worst) price at top, so need to sort
+    [...asks].sort((a, b) => {
+      return b.price - a.price
+    }).forEach((ask) => {
+      const formattedPrice = formatText(ask.price, FORMAT_TYPES.ASK)
+      const formattedDepth = formatText(ask.amount, FORMAT_TYPES.DEPTH)
+      asksTable.push([{ hAlign: 'center', content: formattedPrice }, { hAlign: 'center', content: formattedDepth }])
+    })
+  }
+
+  if (asks.length <= maxLengthPerSide) {
```

Comment:
You should try to unify the logic between asks and bids. Per our conversation, perhaps one way to do this is to treat them identically (e.g. pushing spaces, x more..., etc on the end of the array) until the very end, when you reverse the asks.